### DISCUSSION TITLE
Why the Immersive Translate Team Has Not Added Kodi/Stremio to Their Extension for Translating Movie/Series Subtitles?

### DISCUSSION THREAD

**Discussion by Aslinjean** [immersive-translate/immersive-translate#1835] [General]
The Immersive Translate extension is a remarkable initiative to help users translate movie and series subtitles. However, it is surprising to see that popular platforms like Kodi and Stremio are not yet supported by this extension.

### The Importance of Kodi and Stremio

Kodi and Stremio are extremely popular tools for streaming multimedia content. Their user community is vast and active, as evidenced by the numerous feature requests and discussions on forums like Reddit.

### A Great Opportunity

Adding support for Kodi and Stremio to the Immersive Translate extension would be a great opportunity. It would not only meet the expectations of a large number of users but also enhance the utility and reach of the extension. Many discussions on Reddit highlight users' interest in this feature.

### Call to Action

We encourage the Immersive Translate development team to seriously consider this integration. It would make the tool even more comprehensive and accessible to a wider audience. Adding Kodi and Stremio could transform the user experience by making movies and series more accessible in multiple languages in a more seamless and intuitive manner. 

- I don't know if I will find an answer but help us. dev!@theowenyoung

**Comment by Ricardojarles** [comment]
I completely agree with you, honestly I don't understand why the "Immersive Translate" team hasn't added Kodi/Stremio as the most popular multimedia application on the market, and if Immersive Translate were available on Kodi/Stremio to translate subtitles, I guarantee you that Immersive would be more popular and generate more revenue, because there are few addons that offer this functionality. Most of the addons available on Kodi/Stremio are not available in multiple languages to translate subtitles, which is why Immersive Translate would be a must-have. I am looking forward to the availability of Immersive on Kodi/Stremio.

**Comment by Tipapaof** [comment]
We talk a lot about this topic on Reddit, but unfortunately we can't find any immersive assistant on Reddit. I don't know why they haven't added immersive translation on Reddit, but it would be a good idea to equip the Reddit community. In short... There are many addons available on Stremio to translate subtitles in multiple languages, but they are not effective. And I think that immersive translation would better meet our needs than the available addons, because most of these addons translate subtitles using only Google Translate, which makes the subtitles ineffective anyway... I hope the whole team receives this message and adds Kodi/Stremio to their extensions as soon as possible, because we suffer a lot from not being able to find good addons that meet our needs. If such an initiative becomes a reality, I guarantee the whole team that I will be one of the Immersive Translate users who will pay a monthly subscription.

**Comment by Drakechamp** [comment]
I vote for the entire "immersive translate" team to add their services on Stremio. Honestly, many Stremio users would be thrilled to have such functionality. Since I discovered Stremio, I no longer use Netflix/Disney+. Currently, Stremio has over 60 million users and Kodi has over 70 million. So why aren't these two services available on immersive translate? Maybe the immersive team is not aware of them or they are not interested in adding them to Kodi/Stremio. In my opinion, anyway, I can't wait for immersive to be available on Stremio. I hope the entire team takes your comment into account and makes the services available on the two most popular multimedia platforms in the world.

**Comment by keyboard3** [comment]
Based on my initial research, it appears that these two platforms operate as independent software platforms. Therefore, an immersive browser extension might not be fully capable of accomplishing this task.
